clickreviews/cr_security.py

```diff
@@ -132,6 +132,9 @@ def __init__(self, fn, overrides=None):
             'ubuntu-sdk-15.04': {
                 'policy_version': 1.3,
             },
+            'ubuntu-sdk-16.04': {
+                'policy_version': 16.04,
```

Comment:
shouldn't it be 1.0?

Reply:
@NeoTheThird 16.04 is fine as it references this template https://github.com/ubports/apparmor-easyprof-ubuntu/tree/xenial/data/templates/ubuntu/16.04

Reply:
LGTM :+1: